langchain/document_loaders/trello.py

```diff
@@ -0,0 +1,124 @@
+"""Loader that loads cards from Trello"""
+from typing import List, Optional
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
+
+class TrelloLoader(BaseLoader):
+    """Trello loader. Reads all cards from a Trello board.
+
+    Args:
+        api_key (str): Trello API key.
+        api_token (str): Trello API token.
+    """
+
+    def __init__(self, api_key: str, api_token: str):
+        """Initialize Trello loader."""
+        self.api_key = api_key
+        self.api_token = api_token
+
+    def load(
+        self,
+        board_name: str,
+        card_filter: Optional[str] = "all",
+        include_card_name: Optional[bool] = True,
+        include_comments: Optional[bool] = True,
+        include_checklist: Optional[bool] = True,
+        extra_metadata: Optional[List[str]] = [
```

Comment:
`Sequence` to denote immutability and assign tuple as default value rather than list. No need for an optional.

```suggestion
        extra_metadata: Sequence[str] = ("due_date", "labels", "list", "is_closed")
```